### ISSUE TITLE
feature request: add tf_random_seed argument for TPUConfig

### ISSUE THREAD

**Issue by crystina-z** [tensorflow/tensorflow#37369]
<em>tag:feature_template</em>

**System information**
- TensorFlow version: 1.15 and 2.0
- Are you willing to contribute it (Yes/No): I would like to yet not quite sure how

**Describe the feature and the current behavior/state.**
add `tf_random_seed ` option for `TPUConfig` ([here](https://www.tensorflow.org/api_docs/python/tf/compat/v1/estimator/tpu/TPUConfig)) as for `RunConfig` ([here](https://www.tensorflow.org/api_docs/python/tf/estimator/RunConfig)), so as to control the random seed in tf graph (and also the data sampling process?)

**Will this change the current api? How?**
only one more argument will be added for `TPUConfig`

**Who will benefit with this feature?**
people who want deterministic results on TPU training. 

This feature request comes from the trials to following [this paper](https://arxiv.org/abs/2002.06305): wanna see if bert finetuning also get affected a lot on the other datasets while wanna see the result quickly (on GPU it takes days to train one fold)

**Comment by rmothukuru** [comment]
@crystina-z,
Sorry for the delayed response. The API, [tf.estimator.RunConfig](https://www.tensorflow.org/api_docs/python/tf/estimator/RunConfig) has the argument, **`tf_random_seed`**. Can you please let us know if this is what you are looking for? Thanks!

**Comment by crystina-z** [comment]
@rmothukuru 
Hi! Yea I guess I didn't realize when using TPU, the `RunConfig` could be used as usual..thanks for the response!
